tools/automation/src/pr-check-icons-extensions-404.ts

```diff
@@ -0,0 +1,94 @@
+/********************************************************************************
```

Comment:
Maybe we need an entry point/ main program with flags
Because it's not natural to share source folders with different main programs.

something more natural is like
```
./tool check-icons
./tool create-report
```

Reply:
Maybe this belongs in its own node project under `/.ci/pr-checks/`?